src/StreamStats.jl

```diff
@@ -1,15 +1,15 @@
 module StreamStats
     import StatsBase, Distributions
+    import Base.mean, Base.std, Base.var, Base.cov, Base.cor
+    import Base.maximum, Base.minimum
 
-    export update!, state, nobs, rand, ci
+    export update!, state, nobs, rand, ci, sample
```

Comment:
I didn't mean to imply that this package should have to rely on 'HypothesisTests'  -- if people see a chance of using both packages at the same time, or a third stats package exporting `ci`, then it would seem to be time to bite the bullet and push `ci` into a common dependency like Distributions or StatsBase, but I don't think Julia can do that without adding an actual method in the dependency package at this point. 

Someone with a better idea of the direction of the JuliaStats ecosystem would be better suited to make any decision -- I just wanted to ask the question.